compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeBuilder.java

```diff
@@ -5493,7 +5523,13 @@ private BLangLiteral createSimpleLiteral(Node literal, boolean isFiniteType) {
 
         String textValue;
         if (literal instanceof BasicLiteralNode) {
-            textValue = ((BasicLiteralNode) literal).literalToken().text();
+            Token token = ((BasicLiteralNode) literal).literalToken();
+
```

Comment:
We can remove this empt line.

Reply:
Done in https://github.com/ballerina-platform/ballerina-lang/pull/37471/commits/593be6aecdfc05ddfcc2dabe3b2674b98ab4e6f5#diff-e960b3c3a4848ff505626da90f021ee41db746cf49dcf6e6d69e56f84b23ad36L5534 which will be merged via https://github.com/ballerina-platform/ballerina-lang/pull/37471.